### ISSUE TITLE
While scrolling through a resultset rows go blank

### ISSUE THREAD

**Issue by jashwood** [microsoft/azuredatastudio#18803]
Issue Type: <b>Bug</b>

Nice tool - but it can't do the first thing we need it to - show results from a query. 
When i start scrolling through results using the mouse wheel half of the visible rows turn blank. 

Azure Data Studio version: azuredatastudio 1.35.0 (e9c8eb8564f537726b36b5f15c7f35e62f229284, 2022-02-23T17:36:02.883Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.61GB (4.85GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
agent|Mic|0.48.0
powershell|ms-|2021.12.0
query-history|Mic|0.1.0
schema-compare|Mic|1.12.0
server-report|Mic|0.2.3
whoisactive|Mic|0.1.4
poor-sql-formatter|WSR|0.1.0

</details>

**Comment by alanrenmsft** [comment]
@jashwood  could you please share a gif or screenshot?

**Comment by kburtram** [comment]
I'm unable to repro this issue with the provided information.  Please let us know if there is any additional context that would be helpful to determine what is happening here.  There is a known issue with large XML/JSON/BLOB columns that could cause some issues rendering the result grid while scrolling due to transferring data in the virtualized window https://github.com/microsoft/azuredatastudio/issues/304 that may be related.

**Comment by allan-stewart** [comment]
FWIW, I have had the same problem. It seems to be related to larger datasets (multiple joins). I am selecting 109 columns and none of them are XML, JSON, or BLOB. Some might be VARCHAR(MAX) though? This is querying SqlServer on Amazon RDS.

When I first run the query, there are only 19 rows, but it consistently refuses to display rows 15-19.
![image](https://github.com/microsoft/azuredatastudio/assets/1467577/18c3f34c-6545-4060-8c1e-02b3423478eb)

Then when I scroll all data vanishes.
![image](https://github.com/microsoft/azuredatastudio/assets/1467577/76c468cd-7a20-4979-866f-59bd3e490a77)

**Comment by cheenamalhotra** [comment]
@allan-stewart 

What version of ADS are you using? Is this behavior reproducible with the latest stable v1.46.1?

**Comment by allan-stewart** [comment]
Yes. It happened to me yesterday (when I posted) and I'm on v1.46.1.
